packages/ui-grid/src/GridCol/styles.ts

```diff
@@ -111,31 +113,29 @@ const generateStyle = (
     return breakpoints.slice(breakpoints.indexOf(startAt!))
   }
 
-  const breakpointIsEnabled = (breakpoint: GridBreakpoints) => {
+  const breakpointIsEnabled = (breakpoint: BreakPoints) => {
     return enabledBreakpoints().includes(breakpoint)
   }
 
-  const getColSize = (breakpoint: GridBreakpoints) => {
+  const getColSize = (breakpoint: BreakPoints) => {
     let { width } = props
 
     if (!width) return
 
     if (width && typeof width === 'object') {
-      // @ts-expect-error TODO: type when typing component
-      width = width[breakpoint]
+      width = width[breakpoint === 'x-large' ? 'xLarge' : breakpoint]
```

Comment:
was this a bug that you found while doing the types?

Reply:
Yeah, it seems so, but there were no complaints about it before.

Reply:
I didn't think it was important to mention as a "fix" commit and show it in the Changelog, do you think I should?